recipes/pymol-open-source/1aki.cif

```diff
@@ -0,0 +1,2192 @@
+data_1AKI
```

Comment:
It is possible to download this file from somewhere or take it from the source tarball?

(Mostly concerned with adding a relatively large file into the staged-recipes git history)

Reply:
The file is now downloaded in the test itself.